.circleci/config.yml

```diff
@@ -3,10 +3,12 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: golang:1.12
+      - image: golang:1.13
     steps:
       - checkout
-      - run: go get -u github.com/raviqqe/liche
+        # Do hacky things to install a tool with go modules.
```

Comment:
is: Do hacky things to install a tool with go modules.  
change to: Use unconventional methods to install a tool with Go modules.

Reply:
This is for developers only -- it is not user-facing.

Reply:
Also @bjsignalfx this shouldn't even be in this PR.  Make sure you have rebased from the latest master.

Reply:
@keitwb I branched off release then cherry picked my changeset from master.